common/blocks.py

```diff
@@ -444,6 +444,12 @@ class SimpleIncidentSet(blocks.StructBlock):
         page_type='common.CategoryPage',
         help_text='If selected, only incidents in the chosen category will be included.',
     )
+    categories = blocks.ListBlock(blocks.PageChooserBlock(
+        label='Filter by Category',
+        required=False,
+        page_type='common.CategoryPage',
+        help_text='If selected, only incidents in the chosen category will be included.',
```

Comment:
Sorry to keep mentioning this, but I do think it would be helpful for users of this feature to explicitly spell out in the help text what happens if two or more categories are chosen: for an incident to be included in the chart does it have to be in _all_ the chosen categories, or is any single one of them sufficient?

Reply:
oh sorry this is on my for forgetting, thank you for the reminder!